src/main/kotlin/no/nav/syfo/melding/kafka/KafkaDialogmeldingFraBehandlerConsumer.kt

```diff
@@ -55,14 +56,59 @@ fun processConsumerRecords(
 ) {
     database.connection.use { connection ->
         consumerRecords.forEach {
+            COUNT_KAFKA_CONSUMER_DIALOGMELDING_FRA_BEHANDLER_READ.increment()
             val kafkaDialogmeldingFraBehandler = it.value()
-            log.info("Received a dialogmelding from behandler: navLogId: ${kafkaDialogmeldingFraBehandler.navLogId}, kontorOrgnr: ${kafkaDialogmeldingFraBehandler.legekontorOrgNr}, msgId: ${kafkaDialogmeldingFraBehandler.msgId}")
-            // TODO: Filtrere ut de meldingene vi faktisk skal lagre
+            if (kafkaDialogmeldingFraBehandler != null) {
+                handleIncomingMessage(kafkaDialogmeldingFraBehandler, connection)
+            } else {
+                COUNT_KAFKA_CONSUMER_DIALOGMELDING_FRA_BEHANDLER_TOMBSTONE.increment()
+                log.warn("Received kafkaDialogmeldingFraBehandler with no value: could be tombstone")
+            }
+        }
+        connection.commit()
+    }
+}
+
+private fun handleIncomingMessage(
+    kafkaDialogmeldingFraBehandler: KafkaDialogmeldingFraBehandlerDTO,
+    connection: Connection,
+) {
+    if (kafkaDialogmeldingFraBehandler.msgType == DialogmeldingType.DIALOG_SVAR.name) {
```

Comment:
Denne gir vel `true` også for svar på innkalling til dialogmøte. Hadde det vært lurt å gjøre noe mer sjekk på innholdet her?

Reply:
Kunne gått an å gjøre det ja - hvis det ikke blir for komplekst her. Så slipper man noen unødvendige sjekker i databasen.

Reply:
Tilpasset koden for dette nå 👍

Reply:
Kan vurdere å legge den logikken som sjekker dialog svar forespørsel som extension function på KafkaDialogmeldingFraBehandlerDTO. Så kunne man også sluppet en nøstet if her og gjort COUNT_KAFKA_CONSUMER_DIALOGMELDING_FRA_BEHANDLER_SKIPPED_NOT_DIALOGSVA.increment() én gang.

Reply:
Ja, tror det ble bedre 👍